.github/workflows/molecule.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
         - edpm_nftables
         - edpm_nodes_validation
         - edpm_ovn
+        - edpm_reboot
```

Comment:
i don't think we can test this role here, since these tests are performed in a container. You should take a look at `zuul.d` folder in the repo root for enabling them there (so they can be executed in a vm).

Reply:
moved to zuul.d